apps/web/src/components/terminal/TerminalCard.tsx

```
@@ -0,0 +1,71 @@
+}: TerminalCardProps) {
+  return (
+    <div
+      className={clsx(
+        'flex flex-col h-full min-h-0 min-w-0 rounded-lg overflow-hidden',
+        'border border-border',
+        'bg-card',
+        isFocused && 'ring-2 ring-primary',
+        'transition-all duration-150'
+      )}
+      onClick={onFocus}
+    >
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)